From 58c3e5a5d37cf4e48a3f17fd4b9d192052ff360e Mon Sep 17 00:00:00 2001
From: Ajit Vaishya <ajitv@codeaurora.org>
Date: Tue, 18 Sep 2018 21:55:45 +0530
Subject: [PATCH 15/16] wcnss-service: Add sepolicy to access "vendor.wlan."
 property

Added sepolicy rule so that wcnss-service module can able to write
"vendor.wlan." property.

Change-Id: I457a4bea69dafc174938519bdd9f7c77f87430e9
CRs-Fixed: 2308340
---
 common/wcnss_service.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/wcnss_service.te b/common/wcnss_service.te
index e07d6df..32ccdc0 100644
--- a/common/wcnss_service.te
+++ b/common/wcnss_service.te
@@ -78,3 +78,6 @@ hal_client_domain(wcnss_service, hal_perf)
 
 # read persist.vendor.wigig.npt.enable
 get_prop(wcnss_service, wigig_prop);
+
+# set property to vendor.wlan is enable
+set_prop(wcnss_service, vendor_wifi_prop);
-- 
2.17.1

